scripts/build-video.sh

```diff
@@ -2,6 +2,8 @@
 # Script for building video streaming package
 
 source install/setup.bash
-sudo apt install libopencv-dev -y
-sudo apt install ros-humble-cv-bridge -y
+# sudo apt install libopencv-dev -y
+# sudo apt install ros-humble-cv-bridge -y
```

Comment:
```suggestion
sudo apt install ros-humble-cv-bridge -y
```
Without this package, ros2 command will fail down the line.